backend/tradingbot/tests.py

```diff
@@ -1,11 +1,58 @@
 from django.urls import reverse
-from rest_framework.test import APITestCase
 from rest_framework import status
+from rest_framework.test import APITestCase
+
+from .models import Company, StockTrade
 
 
 class TradingbotTests(APITestCase):
 
+    def setUp(self) -> None:
```

Comment:
This runs before every tests
There is also a special function that will run before the entire test suite which is suitable for other kinds of setup